packages/expo-updates/ios/EXUpdates/Update/EXUpdatesLegacyUpdate.m

```diff
@@ -129,14 +129,7 @@ + (NSURL *)bundledAssetBaseUrlWithManifest:(NSDictionary *)manifest config:(EXUp
     NSString *assetsPathOrUrl = manifest[@"assetUrlOverride"] ?: @"assets";
     // assetUrlOverride may be an absolute or relative URL
     // if relative, we should resolve with respect to the manifest URL
-    NSURL *maybeAssetsUrl = [NSURL URLWithString:assetsPathOrUrl];
-    if (maybeAssetsUrl && maybeAssetsUrl.scheme) {
-      return maybeAssetsUrl;
-    } else if (maybeAssetsUrl && maybeAssetsUrl.standardizedURL) {
-      return [manifestUrl.URLByDeletingLastPathComponent URLByAppendingPathComponent:maybeAssetsUrl.standardizedURL.relativeString];
-    } else {
-      return [manifestUrl.URLByDeletingLastPathComponent URLByAppendingPathComponent:assetsPathOrUrl];
-    }
+    return [NSURL URLWithString:assetsPathOrUrl relativeToURL:manifestUrl.standardizedURL];
```

Comment:
Same comment about adding tests for normalization. There's some chance this needs to return `.standardizedURL` of the resolved URL.